go.mod

```diff
@@ -83,3 +82,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 )
+
+replace github.com/google/go-sev-guest => github.com/edgelesssys/go-sev-guest v0.0.0-20240705062330-4bd4b2483aa0
```

Comment:
Replace should be placed in the top of the file, in one section with existing replaces. Please add a comment why we need the replace, linking to the upstream issue that must be resolved to use the upstream module again.